### ISSUE TITLE
Global top 10 not displaying correctly for Results

### ISSUE THREAD

**Issue by jjmountain** [trouni/predictor-vue#240]
This is what I see if I swipe to global top 10 on results page.

<img width="807" alt="2022-12-07_14-37-52" src="https://user-images.githubusercontent.com/21108437/206097424-94e96589-6316-4704-9a98-960d74ed54e1.png">

Is this even supposed to be available from the results page? It would seem not.

**Comment by trouni** [comment]
yeah, I think we could either hide this if there are no results available, or maybe just show the global percentages (but this would require some work on the BE)
